src/core/views.py

```diff
@@ -3559,14 +3714,143 @@ def view_proposal_review(request, proposal_id, assignment_id, access_key=None):
         'recommendation_form': recommendation_form,
         'book_editors': _proposal.book_editors.all(),
         'active': 'proposal_review',
-        'instructions': get_setting('instructions_for_task_proposal',
-                                    'general'),
+        'instructions': get_setting(
+            'instructions_for_task_proposal',
+            'general'
+        ),
         'data': data,
     }
 
     return render(request, template, context)
 
 
+class ProposalReviewCompletionEmail(FormView):
+    """Allows requested reviewers to send an email to requesting editors
+    after they have responded to a peer review request.
+    """
+    template_name = 'shared/editable_notification_email.html'
+    form_class = forms.CustomEmailForm
+
+    @method_decorator(is_reviewer)
+    def dispatch(self, request, *args, **kwargs):
+        self.proposal_id = self.kwargs['proposal_id']
+        self.assignment_id = self.kwargs['assignment_id']
+        self.access_key = self.kwargs.get('access_key')
+
+        self.proposal = get_object_or_404(
+            submission_models.Proposal,
+            pk=self.proposal_id
+        )
+
+        if self.access_key:
+            self.proposal_reivew = get_object_or_404(
+                submission_models.ProposalReview,
+                pk=self.assignment_id,
+                access_key=self.access_key,
+            )
+        else:
+            self.proposal_reivew = get_object_or_404(
+                submission_models.ProposalReview,
+                pk=self.assignment_id,
+            )
+
+        return super(ProposalReviewCompletionEmail, self).dispatch(
+            request,
+            *args,
+            **kwargs
+        )
+
+    def get_context_data(self, **kwargs):
+        context = super(ProposalReviewCompletionEmail, self).get_context_data(
+            **kwargs
+        )
+        context['heading'] = (
+            'Please ensure that you are happy with the below email to the '
+            'editor notifying them that you have completed your review'
```

Comment:
Shall we end this sentence with a full stop?

Reply:
It's a heading so it might look a bit odd with a full stop.
<img width="1320" alt="screen shot 2018-09-07 at 11 02 00" src="https://user-images.githubusercontent.com/24626568/45212728-88cbf180-b28d-11e8-96fc-55a682270e50.png">